### DISCUSSION TITLE
Puzzle for Figma

### DISCUSSION THREAD

**Discussion by Nevu** [ingrammicro/puzzle-publisher#58] [General]
Hi,

we think about switching to Figma sometimes. Some colleges wish something like Puzzle for Figma (seems there’s no good alternative in Figma).

Do you plan Puzzles for Figma?

Regards

**Comment by MaxBazarov** [comment]
> Hi,
> 
> we think about switching to Figma sometimes. Some colleges wish something like Puzzle for Figma (seems there’s no good alternative in Figma).
> 
> Do you plan Puzzles for Figma?

Yes. Have no any strong ETAs for now, but I'm planning to start it very soon.
BTW. I already developed [one plugin](https://github.com/MaxBazarov/style-station-plugin) for Figma.

**Reply by MaxBazarov** [reply]
@Nevu  If you are still interesting in the subj then I can share new information. I have a working plugin prototype. Hope to release Beta 1 during October.

**Comment by MaxBazarov** [comment]
> Hi,
> 
> we think about switching to Figma sometimes. Some colleges wish something like Puzzle for Figma (seems there’s no good alternative in Figma).
> 
> Do you plan Puzzles for Figma?
> 
> Regards

Just released
https://www.figma.com/community/plugin/1033284895522313541/Demo-Station

**Reply by Nevu** [reply]
I just shared!

I'll give you feedback while evaluate. Thank you very much!
